nifi-nar-bundles/nifi-slack-bundle/nifi-slack-processors/pom.xml

```
@@ -85,5 +85,34 @@
             <artifactId>nifi-ssl-context-service-api</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-record-sink-api</artifactId>
+            <scope>compile</scope>
```

All of the `api` dependencies should be marked as `provided`, instead of `compile`.
```suggestion
            <scope>provided</scope>
```